spark-paper/src/main/java/me/lucko/spark/paper/PaperWorldInfoProvider.java

```diff
@@ -110,11 +111,12 @@ public GameRulesResult pollGameRules() {
     @SuppressWarnings("removal")
     @Override
     public Collection<DataPackInfo> pollDataPacks() {
-        return this.server.getDataPackManager().getDataPacks().stream()
+        this.server.getDatapackManager().refreshPacks();
+        return this.server.getDatapackManager().getPacks().stream()
```

Comment:
Is is best to list all the available ones or just enabled ones?

Reply:
All the available ones since that follows the other datapack impls. Maybe a feature request (or future PR?) could be to show more datapack info that has been exposed through the Datapack API.